pkg/sql/plan2/explain/build_expr_explain.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package explain
+
+import (
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/pb/plan"
+)
+
+func DescribeExpr(expr *plan.Expr) string {
```

Comment:
I think most of your functions is private will be better.

Reply:
I will consider revising this question later